src/createAnimatedComponent/getViewInfo.ts

```diff
@@ -0,0 +1,36 @@
+'use strict';
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+// This is a makeshift solution to handle both 0.73 and 0.74 versions of React Native.
+
+export let getViewInfo = (element: any) => {
+  if (element._nativeTag) {
+    getViewInfo = getViewInfo73;
+    return getViewInfo73(element);
+  } else if (element.__nativeTag) {
```

Comment:
Can `__nativeTag` be equal to `0`? Do we want to check if `__nativeTag > 0` of if exists?

Reply:
good question, I will make it more error-proof